### PR TITLE
Pagetree Fixes

### PR DESCRIPTION
**Pull request by FinalAngel**
- Re-implement shift+click on settings icon (redirect to advanced settings)
- Remove "Pages" title on first column
- Rename "Options" back to "Actions"
- Implement reload link to error messages
- Fix reloadBrowser on iframe (no need to reload the entire project)

### CODE REVIEW

File: cms/static/cms/js/modules/cms.pagetree.js [hunk 1]
```diff
@@ -738,6 +752,21 @@ var CMS = window.CMS || {};
...
+         * need to skip `CMS.API.Helpers.reloadBrowser();`.
+         *
+         * @method _reloadHelper
+         * @private
+         */
+        _reloadHelper: function _reloadHelper() {
+            if (window.self !== window.top) {
+                window.location.reload();
+            } else {
+                CMS.API.Helpers.reloadBrowser();
```
Comment (vxsx) [inline cms/static/cms/js/modules/cms.pagetree.js]: you might want to do `false, false, true` here like in the https://github.com/divio/django-cms/pull/4906/files

File: cms/templates/admin/cms/page/tree/base.html [hunk 2]
```diff
@@ -142,7 +143,7 @@
                     "title": "{% trans 'Menu' %}",
                     "key": "menu"
                 }, {
-                    "title": "{% trans 'Options' %}",
+                    "title": "{% trans 'Actions' %}",
```
Comment (vxsx) [inline cms/templates/admin/cms/page/tree/base.html]: btw, we should probably do `{{ _('Actions')|escapejs }}` for this kind of thing (not paranoid)

File: cms/static/cms/js/modules/cms.pagetree.js (hunk 1 above)
Comment (FinalAngel) [inline cms/static/cms/js/modules/cms.pagetree.js]: This is handled separately here: https://github.com/FinalAngel/django-cms/blob/feature/adaptions/cms/static/cms/js/modules/cms.pagetree.js#L682

**Comment by vxsx** [comment]
:+1:

**Pull request merged**